### DISCUSSION TITLE
firefox: cannot access keepassxc with addon (AppImage)

### DISCUSSION THREAD

**Discussion by nidamanx** [netblue30/firejail#4063] [Q&A]
KeePassXC updates are coming in AppImage format. I'm testing some firejail params to enable communication between KeePassXC-Browser (official KeePassXC plugin for browsers) and KeePassXC database.
All is fine if KeePassXC is from Debian repo but, with KeePassXC as AppImage, I cannot find a way to have a working plugin.
So, my question is: 
is there a way to debug firefox profile so I can see which param(s) I need to enable/add to the profile?
BTW, `firejail --noprofile firefox` works well but, who wants to use a browser without this great sandbox?
Thanks!

**Comment by reinerh** [comment]
Sorry, I can't really help you with your problem, as I have no experience with AppImage.
I'm just wondering why you want to use an AppImage for KeePassXC instead of the native Debian package?

**Reply by nidamanx** [reply]
Thanks @reinerh
I just wanted to have a firejail profile compatible with AppImage too. So in case someone will need for who knows which reason, it's already done.

**Reply by rusty-snake** [reply]
> I just wanted to have a firejail profile compatible with AppImage too.

I don't think that 100% possible because AIs can be everywhere with any name.

**Comment by rusty-snake** [comment]
> firejail --noprofile firefox works well but, who wants to use a browser without this great sandbox?

What happens if you `whitelist` the AI?

**Reply by nidamanx** [reply]
Sorry. I gave up after many tries. The best is to use KeePassXC from the Debian Repository

**Reply by euidzero** [reply]
Hello,
 
 If I understand well you managed to acheive a firejail firefox communcating with a non firejail keepassxc.
Did you had to modifiy the firejail profile for firefox ? Could you post your modifications ?

THX

**Reply by rusty-snake** [reply]
https://github.com/netblue30/firejail/blob/efbf74e12421c97d8a1756649422f83f4a0b7e50/etc/profile-a-l/firefox.profile#L28-L31

**Reply by nidamanx** [reply]
> Hello,
> 
> If I understand well you managed to acheive a firejail firefox communcating with a non firejail keepassxc. Did you had to modifiy the firejail profile for firefox ? Could you post your modifications ?
> 
> THX

My idea was to integrate Firefox and KeePassXC from any sources using one .profile file
I had problems with the AppImage so I decided to skip.
Firefox .local profile needs some editing in case you want to use customized places for KeePassXC files (check .profile in /etc/firejail to see how to customize the parameters)

**Reply by rusty-snake** [reply]
#3200 and #3952

**Reply by euidzero** [reply]
Haha, just found it in the meantime (I was looking for the issue with brave so I did not come accross the firefox config). Many thx !
